src/main/java/com/rackspace/salus/telemetry/model/JobType.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.salus.telemetry.model;
+
+public enum JobType {
+  DELETE_TENANT("Delete tenant");
```

Comment:
```suggestion
  DELETE_TENANT("Delete Tenant");
```

I hesitated on being this picky last night, but the lowercase `t` is still bugging me a little :)